packages/framework/src/Resources/views/Admin/Layout/displayUuid.html.twig

```diff
@@ -0,0 +1,6 @@
+{% set keys = _context|keys %}
+{% if (keys[1] is defined) and (attribute(_context[keys[1]], 'uuid') is defined) %}
+    <span class="edit-uuid">
+        {{ _context[keys[1]].uuid }}
+    </span>
+{% endif %}
```

Comment:
It is weird to me to have one of many attributes of an entity displayed somewhere else than others, but I feel it is not my decision - @pk16011990

Reply:
I agree, It's weird. However, I understand that it's managed from one place.
Idea: Replace it with the text "UUID" with the tooltip (shows UUID hash) and icon "Copy to clipboard". And with a smaller margin-left, e.g. 1em.

Reply:
Well, the UX of the administration is far from ideal, and displaying more fields in the already long enough list of attributes certainly does not help. Especially when some of these fields are for the common administrator irrelevant (ID, UUID)

I like the idea of the icon – changed as you described, now it's quite elegant in my opinion :)

Reply:
My point still stands. It is nice that it is managed from a single point, but I would prefer to add it to each form. 

The argument that the current UX is not ideal is valid, but adding a new flowing element is in my opinion less perfect than adding an attribute to an already long list, even if the administrator does not need it.